src/utils/dateUtils.js

```diff
@@ -20,6 +16,18 @@ export const getDateRange = (startDate, endDate) => {
   return dateRange;
 };
 
+/**
+ * Converts Date objects to match the date picker's
+ * format. Basically the date picker just defaults
+ * dates to local date, 12pm.
+ *
+ * @param {Object} date
+ */
+export const convertToPickedDate = (date) => {
```

Comment:
convertToPickedDate - maybe: setToBeginningOfDay

Also, your convertToPickedDate() modifies the argument -- this is really bad style. This is better:
```
export const convertToPickedDate = (date) => {
  let beginning_of_day = new Date(date);
  beginning_of_day.setHours(12, 0, 0, 0);
  return beginning_of_day;
};
```
(There might be a better way of doing this; my JS-fu is not good.)

Reply:
done, still thinking about the function name tho, i might end up changing the functionality...